src/ApplicationLogs/LogReader.cs

```diff
@@ -34,12 +35,16 @@ public LogReader()
         {
             Blockchain.Committing += OnCommitting;
             Blockchain.Committed += OnCommitted;
+            if (Settings.Default.Debug)
+                ApplicationEngine.Log += ApplicationEngine_Log;
```

Comment:
It's a nice feature. How about adding the same ability to add logs to execution result for `invokescript`, `invokefunction` and etc. in debug mode? This change is more invasive and requires changing the core library as far, but it might be useful for someone.

Reply:
@AnnaShaleva Also https://github.com/neo-project/neo-modules/pull/807#issuecomment-1807590923